stdlib/experimental/query/from.flux

```diff
@@ -24,3 +26,30 @@ inBucket = (
         |> filterMeasurement(measurement)
         |> filter(fn: predicate)
         |> filterFields(fields)
+
+splitData = (v, t=",") => strings.split(v:v, t:t)
+sanitizeList = (v) => strings.replaceAll(v:v, t: ", ", u: ",")
+
+select = (
+    src,
+    fields,
+    start,
+    stop=now(),
+    where=(r) => true
```

Comment:
I don't think this adds anything to the query since it is easily replicated with adding `|> filter()` after the query which is more composable.